### ISSUE TITLE
Rework SET command

### ISSUE THREAD

**Issue by 1st1** [edgedb/edgedb#347]
Proposed:

```
SET CONFIG foo := 1;
SET GLOBAL var := 'aaa';
SET MODULE my_foo;
SET ALIAS bar AS MODULE mybar;

RESET CONFIG foo;
RESET GLOBAL var;
RESET MODULE;
RESET ALIAS bar;
RESET ALIAS *;
```

**Comment by elprans** [comment]
Using globals in queries:

```
CREATE GLOBAL my::foo;
SELECT (GLOBAL my::foo);
```

Using config values in queries:
```
SELECT
sys::Config.foo
```

where `sys::Config` is a special view containing all configuration variables.

**Comment by elprans** [comment]
Fixed in #354

**Comment by 1st1** [comment]
With the "CONFIGURE" command implemented I think this issue is safe to close now.
